packages/remark-parse/types/index.d.ts

```diff
@@ -24,7 +27,7 @@ declare namespace remarkParse {
   interface RemarkParseOptions {
     /**
      * GFM mode
-     *
+     *RemarkParser
```

Comment:
this is unrelated to the comment, but does seem to be related to the missing class name above.
Please remove this from here, and add it back to the class.
Thanks!

Reply:
i am sorry for this dumb mistaken. 

I have removed uncessary part.